OxfordBodleian/Juel-Jensen/BDLaethd23.xml

```diff
@@ -218,12 +505,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                     </physDesc>
 
-                <!--    <history>
+                <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <origDate when="1700" evidence="1799">eighteenth century</origDate>
+                            According to <ref target="#a4"/> <!--check if correct-->, the manuscript was produced
```

Comment:
`<ref target="#a4">Additional note 4</ref>`

Reply:
Varia note X

Reply:
What's the reference here? Remember we cannot cite the booklet!

Reply:
References to different parts of the record. For example, when talking about the dating, one may want to include something like "see Additional note 1" as evidence.